src/widgets/WidgetMainQmc.vue

```diff
@@ -129,16 +129,15 @@ export default defineComponent({
     },
   },
   created() {
-    this.getPrimaryProduct()
+    this.setProduct()
     this.getCardAccountFee()
     this.getCardStandardInterestRate()
     this.createTitle()
     this.createSubtitle()
   },
   methods: {
-    getPrimaryProduct() {
-      this.primaryProduct.productType = this.products[0].productType
-      this.primaryProduct.productItems = this.products[0].productItems
+    setProduct() {
+      this.displayedProduct = this.products[0]
```

Comment:
Setting the product has been moved from `DialogOverlay.vue` to the widget
The reason I decided to make a function out of it, is because I think this is more readable, and in the future, the user might be able to switch products (for now, we are expected to show only the first product in the array)